azure-governance-cloudwise/azuredeploy.json

```diff
@@ -0,0 +1,573 @@
+      "type": "string",
+      "defaultValue": "East US",
+      "allowedValues": [
+        "Australia Southeast",
+        "East US",
+        "Southeast Asia",
+        "West Europe"
+      ],
+      "metadata": {
+        "description": "Location of the OMS regions."
+      }
+    },
+
+    "_artifactsLocation": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/AvyanConsultingCorp/azure-quickstart-templates/master/azure-governance-cloudwise",
+      "metadata": {
+        "description": "Location for the artifacts"
+      }
+    },
+    "_artifactsLocationSasToken": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Reserved for deploying using Visual Studio. Please keep it as an empty string"
+      }
+    }
+  },
+  "variables": {
+    "packageURI": "https://msignite2016stg.blob.core.windows.net/cloudwise/package.zip", 
+
+    "WebAppDeployPackageFolder": "ISVDemoUsage",
+    "WebAppDeployPackageFileName": "package.zip",
+    "hostingPlanName": "[concat('CloudWiseHostingplan-', uniqueString(resourceGroup().id))]",
+    "webSiteName": "[concat('CloudWise', uniqueString(resourceGroup().id))]",
+    "sqlserverName": "[concat('sql',parameters('sqlserverName'), uniqueString(resourceGroup().id, deployment().name))]",
+
+    "insightsName": "[concat(variables('webSiteName'),'insights')]",
+    "omsWorkspaceName": "[concat(parameters('omsWorkspaceName'), uniqueString(resourceGroup().id, deployment().name))]",
+
+    "OMSWebAppsRunBookAndDashboardTemplateFolder": "nested/AutomationRunbooksOMSDashboard",
+    "OMSWebAppsRunBookAndDashboardTemplateFileName": "OMSAutomationRunbooksDashboard.json",
+
+    "OMSCommonTemplateFolder": "nested/OMSCommon",
+    "OMSCommonTemplateFileName": "OMSCommon.json"
+  },
+  "resources": [
+    {
+      "name": "[variables('sqlserverName')]",
+      "type": "Microsoft.Sql/servers",
+      "location": "[resourceGroup().location]",
+      "tags": {
+        "displayName": "SqlServer"
+      },
+      "apiVersion": "2014-04-01-preview",
+      "properties": {
+        "administratorLogin": "[parameters('sqlAdministratorLogin')]",
+        "administratorLoginPassword": "[parameters('sqlAdministratorLoginPassword')]"
+      },
+      "resources": [
+        {
+          "name": "[parameters('sqlDatabaseName')]",
+          "type": "databases",
+          "location": "[resourceGroup().location]",
+          "apiVersion": "2014-04-01-preview",
+          "dependsOn": [
+            "[concat('Microsoft.Sql/servers/', variables('sqlServerName'))]"
+          ],
+          "properties": {
+            "edition": "[parameters('sqlEdition')]",
+            "collation": "[parameters('sqlCollation')]",
+            "maxSizeBytes": "[parameters('sqlMaxSizeBytes')]",
+            "requestedServiceObjectiveName": "[parameters('sqlRequestedServiceObjectiveName')]"
+          }
+        },
+        {
+          "apiVersion": "2014-04-01-preview",
```

Comment:
Couldn't find the latest SQL DB schemas.
1) Looked @ https://github.com/Azure/azure-resource-manager-schemas
2) as well as scanned the quickstarts repository.

All projects use "2014-04-01-preview"

Please advise if there's a latest one.